index.html

```diff
@@ -0,0 +1,237 @@
+                </li>
+                <li>
+                    <h2>My page</h2>
+                </li>
+                <li>
+                    <h2>Logout</h2>
+                </li>
+            </ul>
+        </div>
+
+        <div class="desktop-navbar">
+            <div class="desktop-logo">
+                <a href="#"><img src="./images/download.jpg" alt=""></a>
+            </div>
+            <div class="navbar-pages">
+                <ul>
+                    <li><a href="about.html">About</a></li>
+                    <li><a href="">Program</a></li>
+                    <li><a href="">Join</a></li>
+                    <li><a href="">Sponsor</a></li>
+                    <li><a href="">News</a></li>
+                    <li class="navbar-button" id='on'>CC Campaign</li>
+                </ul>
+            </div>
+        </div>
+
+        <!-- mobile navbar -->
+
+        <div class="navbar-container">
+
+            <div class="nav-logo">
+                <a href="#"><img class="back-image" src="./images/hamburger.png" onclick=showMenu() id='menu-bar'
+                        alt="menu"></a>
+            </div>
+            <div class='main-menu'>
+                <div id='close'>
+                    <img src="./images/close.png" onclick=close() alt="close">
+                </div>
+
+
+                <ul class="main-links">
+                    <li>
+                        <a href="about.html">About</a>
+                    </li>
+                    <li>
+                        <a href="#program">Program</a>
+                    </li>
+                    <li>
+                        <a href="#join">Join</a>
+                    </li>
+                    <li>
+                        <a href="#sponsor">Sponsor</a>
+                    </li>
+                    <li>
+                        <a href="#News">News</a>
+                    </li>
+                    <li>
+                        <a href="#projects" class='highlight' id='on'>projects</a>
+                    </li>
+                </ul>
+            </div>
+        </div>
+        </div>
+
+
+    </navbar>
+
+    <section class="headline">
+        <div class="headline-container">
+            <div class="headline-subtitle">
+                <h2>"Hello, welcome to"</h2>
+
+            </div>
+            <div class="headline-title">
+                <h1>AFRICAN MOVIES<br />FESTIVAL<br />2023</h1>
+            </div>
+            <div class="headline-paragraph">
+                <p>
+                    Africa Movies was established in 2020 to bring on-board the
+                    African youth to the Movie industry. We welcome you for our
+                    third screening event.
+                </p>
+            </div>
+            <div class="headline-time">
+                <h3>2023.1.04(FRI) ~ 2(SAT)</h3>
+            </div>
+            <div class="headline-venue">
+                <p>Kigali arena, Kigali,Rwanda</p>
+            </div>
```

Comment:
- [ ] @kazaneza great work implementing the changes requested however when doing so the design got broken, kindly ensure this section doesn't overlap with the program section
![image](https://user-images.githubusercontent.com/61408860/212413930-9433567c-a38f-42cb-9752-82cef3d79971.png)